phac-nml/irida-next
# Project change path

## What does this PR do and why?
Adding `Change path` section under project `Advanced Settings` like described from #76.

## Screenshots or screen recordings
![image](https://github.com/phac-nml/irida-next/assets/325703/278e036c-aa8b-4c16-8f43-4fe6c22bbe7f)

## How to set up and validate locally
1. Navigate to a project settings page.
2. Update path field under `Advanced Settings` in the `Change project URL` section.
3. Hit the `Change path` button.
4. Verify the path changed.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

  the [PR acceptance checklist](https://phac-nml.github.io/irida-next/docs/development/development_processes/code_review#acceptance-checklist) for this PR.
